ANSWERS.md

```diff
@@ -1,7 +1,15 @@
 <!-- Answers to the Short Answer Essay Questions go here -->
 
 1. In Mocha, what are the differences between `before`, `after`, `beforeEach`, and `afterEach`? When do they run? What are they used for?
+Before: Actions that happen before a block occurs
+After: Actions that happen after a block occurs
+They are used to set up testing
+

```

Comment:
These may be true statements, but these answers do not demonstrate your depth of understanding. What is the difference between `before` and `beforeEach`? "To set up testing"? Sure, but why is this useful? (Hint: D.R.Y.)